### PR TITLE
Update process.md and README.md

### PR DESCRIPTION
**Pull request by theresalech**
Updated process.md to include references to the SDLC and add proposal decision types.  Updated README.md to include platform descriptions and recent/upcoming release information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,35 +19,116 @@ Important documents:
...
+#### iOS SDK: 4.7.0
+**Expected Release Date**: August 31, 2017
+
+**Contents**: Bug fixes and new features included on the [iOS 4.7.0 Project Page](https://github.com/smartdevicelink/sdl_ios/projects/5).
+
+
+#### SDL Protocol: 5.0.0
+**Expected Release Date**: October 30, 2017
+
+**Contents**: Introducing control frame payloads and a revised version negotiation scheme.  See the current draft of the new version of the spec [here](https://github.com/smartdevicelink/protocol_spec/pull/9).
```
Comment (joeljfischer) [inline README.md]: Control frame payloads and revised scheme are actually in v4.7

File: process.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+Any changes to the [SDL RPC spec](https://github.com/smartdevicelink/rpc_spec), the [SDL protocol](https://github.com/smartdevicelink/protocol_spec), public API changes to the SDL [iOS](https://github.com/smartdevicelink/sdl_ios) or [Android](https://github.com/smartdevicelink/sdl_android) SDKs, or major changes to [SDL Core](https://github.com/smartdevicelink/sdl_core) must go through the SDL evolution proposal and review process. SDL is versioned using [semantic versioning](http://www.semver.org). Therefore, any public API removal, alteration, or addition is a minor or major version change to the respective spec or library, no matter how small the change. The only minor or major change that does not need to go through this process is a misspelled API name in a mobile SDK. In that case, write an issue against the respective repository. Changes that do not make a spec or public API change should go through the normal contribution process, for example, [this is the iOS repository's process](https://github.com/smartdevicelink/sdl_ios/blob/master/.github/CONTRIBUTING.md).
```
Comment (joeljfischer) [inline process.md]: I think we actually agreed on an enhancement / bug fix scheme over the API change scheme. Some changes may not actually change the API surface but will need proposals. It's an enhancement if it changes the way developers use the project, and a bug fix if it changes it, but changes it to the way it was supposed to work, but failed to do so.

File: process.md [hunk 3]
```diff
@@ -23,25 +23,33 @@ Please state explicitly whether you believe that the proposal should be accepted
...
 * **Develop the proposal**: expand the rough sketch into a complete proposal, using the [proposal template](0000-template.md), and continue to refine the proposal on the evolution slack channel. Prototyping an implementation and its uses along with the proposal is encouraged, because it helps ensure both technical feasibility of the proposal as well as validating that the proposal solves the problems it is meant to solve.
 * **Request a review**: initiate a pull request to the [sdl_evolution repository][sdl_evolution_repo] to indicate to the maintainers that you would like the proposal to be reviewed. When the proposal is sufficiently detailed and clear, and addresses feedback from earlier discussions of the idea, the pull request will be accepted. The proposal will be assigned a proposal number as well as a maintainer to manage the review. In addition, an issue will be opened specifically for the review and attached to the proposal.
 * **Address feedback**: in general, and especially [during the review period](#review), be responsive to questions and feedback about the proposal.
 
 ## Review process
```
Comment (joeljfischer) [inline process.md]: Some headers have a blank line, others don't. It should be standardized.

Comment (joeljfischer) [inline process.md]: I would recommend a blank line before the header and none after.

File: README.md (hunk 1 above)
Comment (theresalech) [inline README.md]: Confirmed that these are in Protocol 5.0.0 and iOS 4.7.0.

**Pull request merged**